boards/st/stm32mp135f_dk/stm32mp135f_dk.dts

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright (c) 2025 STMicroelectronics
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+#include <st/mp13/stm32mp135.dtsi>
+
+#include <zephyr/dt-bindings/input/input-event-codes.h>
+#include <zephyr/dt-bindings/pinctrl/stm32-pinctrl.h>
+
+/ {
+	model = "STMicroelectronics STM32MP135-DK board";
+	compatible = "st,stm32mp135f-dk";
+	chosen {
+		zephyr,flash = &ddr_code;
+		zephyr,sram = &ddr_data;
+		zephyr,console = &uart4;
+		zephyr,shell-uart = &uart4;
+	};
+
+	gpio_keys {
+		compatible = "gpio-keys";
+		user_button: button {
+			label = "User 1";
+			gpios = <&gpioa 14 GPIO_ACTIVE_LOW>;
+			zephyr,code = <INPUT_KEY_0>;
+		};
+	};
+
+	leds {
+		compatible = "gpio-leds";
+		blue_led_1: led_1 {
+			gpios = <&gpioa 14 GPIO_ACTIVE_HIGH>;
+			label = "LD3";
+		};
+		red_led_2: led_2 {
+			gpios = <&gpioa 13 GPIO_ACTIVE_HIGH>;
+			label = "LD4";
+		};
+	};
+
+	aliases {
+		led0 = &blue_led_1;
+		led1 = &red_led_2;
+		sw0 = &user_button;
+	};
+};
+
+&clk_hsi {
+	status = "okay";
+};
+
+&clk_hse {
+	status = "okay";
+};
+
+&pinctrl {
```

Comment:
Include and use a -pinctrl.dtsi file that you generated for hal_stm32 PR (https://github.com/zephyrproject-rtos/hal_stm32/pull/266/files#diff-6ddf6e4725bd5ca9a5361de2a3ec62cdaf6a48dd6b4b655cb3ffcc480b492476)